src/main/java/com/cb/services/dbService/dbServiceImpl/CharacterDBServiceImpl.java

```diff
@@ -1,7 +1,7 @@
-package com.cb.services.DBService.DBServiceImpl;
+package com.cb.services.dbService.dbServiceImpl;
 
 import com.cb.dal.CharacterDAL;
-import com.cb.services.DBService.IDBService.CharacterDBService;
+import com.cb.services.dbService.iDbService.CharacterDBService;
```

Comment:
I think naming standart is first capital letter "I" instead of "i"

Reply:
naming standard for folders is first letter - lowercase.